src/js/sensor_types.js

```diff
@@ -95,7 +95,7 @@ export function sensorTypes() {
         },
         sonar: {
             name: "Sonar",
-            elements: ["NONE", "HCSR04", "TFMINI", "TF02", "MTF01", "MTF02", "MTF01P", "MTF02P"],
+            elements: ["NONE", "HCSR04", "TFMINI", "TF02", "MTF01", "MTF02", "MTF01P", "MTF02P", "TFNOVA"],
```

Comment:
Only suggestion maybe to put this line in another PR (adding TFNOVA - which requires firmware update)
```suggestion
            elements: ["NONE", "HCSR04", "TFMINI", "TF02", "MTF01", "MTF02", "MTF01P", "MTF02P"],
```

Reply:
ok